app/Views/admin/products/Products.php

```diff
@@ -0,0 +1,122 @@
+                                <i class="ri-add-circle-fill label-icon align-middle rounded-pill fs-16 me-2"></i>Agregar Producto
+                            </a>
+                        </div>
+                    </div>
+
+                    <?php
+                        if (session()->has('response')) {
+                            $response = session()->get('response');
+                            ?>
+                        <div id="alertElement" data-response="<?= htmlspecialchars(json_encode($response)) ?>"></div>
+                    <?php } ?>
+
+                    <!-- card-header -->
+                    <div class="card-body">
+                        <table id="products-table" class="table nowrap dt-responsive align-middle table-hover table-bordered w-100">
+                            <thead>
+                                <tr>
+                                    <th>ID</th>
+                                    <th>Nombre</th>
+                                    <th>Modelo</th>
+                                    <th>Categoría</th>
+                                    <th>Imagen</th>
+                                    <th>Acciones</th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                <?php foreach ($products as $product) { ?>
+                                    <tr>
+                                        <td><?= $product['productId']; ?></td>
+                                        <td>
+                                            <div class="d-flex align-items-center fw-medium">
+                                                <p class="text-wrap"><?= $product['productName'] ?></p>
+                                            </div>
+                                        </td>
+                                        <td>
+                                            <div class="d-flex align-items-center fw-medium">
+                                                <p class="text-wrap"><?= $product['productModel'] ?></p>
+                                            </div>
+                                        </td>
+                                        <td>
+                                            <div class="d-flex align-items-center fw-medium">
+                                                <p class="text-wrap"><?= $product['categoryName'] ?></p>
+                                            </div>
+                                        </td>
+                                        <td>
+                                            <div class="d-flex justify-content-center align-items-center">
+                                                <img class="mw-100 object-fit-contain" src="<?= $product['fileRoute']; ?>" alt="Imagen del certificado " . <?= $product['productId']; ?> height="150">
```

Comment:
```suggestion
                                                <img class="mw-100 object-fit-contain" src="<?= $product['fileRoute']; ?>" alt="<?= 'Imagen del certificado' . $product['productId']; ?>" height="90">
```

Syntax error and can we reuce please the height of the image please. 🙏
To be honest i think maybe we should reduce the size of other table entities too.